AutoMaxLairUI - Code/Form1.cs

```diff
@@ -0,0 +1,614 @@
+            t["TESSERACT_PATH"] = boxTesseract.Text;
+
+            // Advanced Settings
+            t["advanced"]["VIDEO_SCALE"] = float.Parse(boxVideoScale.Text);
+            t["advanced"]["VIDEO_EXTRA_DELAY"] = float.Parse(boxVideoDelay.Text);
+            t["advanced"]["BOSS_INDEX"] = boss;
+            t["advanced"]["DYNITE_ORE"] = int.Parse(boxDyniteOre.Text);
+            t["advanced"]["CONSECUTIVE_RESETS"] = int.Parse(boxConsecutiveResets.Text);
+            t["advanced"]["ENABLE_DEBUG_LOGS"] = checkBoxDebugLogs.Checked;
+
+            // Stat Settings
+            t["stats"]["CHECK_ATTACK_STAT"] = boxCheckAttack.Checked;
+            t["stats"]["CHECK_SPEED_STAT"] = boxCheckSpeed.Checked;
+
+            // Create TomlNodes for each stat +,=,-
+            TomlNode node = new TomlNode[] { };
+            TomlNode node1 = new TomlNode[] { };
+            TomlNode node2 = new TomlNode[] { };
+            TomlNode node3 = new TomlNode[] { };
+            TomlNode node4 = new TomlNode[] { };
+            TomlNode node5 = new TomlNode[] { };
+
+            // Get all values from the array into int and add it to the node
+            for (int i = 0; i < atkPos.Length; i++)
+            {
+                
+                node.Add(int.Parse(atkPos[i]));
+                
+            }
+            for (int i = 0; i < atkNeut.Length; i++)
+            {
+
+                node1.Add(int.Parse(atkNeut[i]));
+
+            }
+            for (int i = 0; i < atkNeg.Length; i++)
+            {
+
+                node2.Add(int.Parse(atkNeg[i]));
+
+            }
+            for (int i = 0; i < speedPos.Length; i++)
+            {
+
+                node3.Add(int.Parse(speedPos[i]));
+
+            }
+            for (int i = 0; i < speedNeut.Length; i++)
+            {
+
+                node4.Add(int.Parse(speedNeut[i]));
+
+            }
+            for (int i = 0; i < speedNeg.Length; i++)
+            {
+
+                node5.Add(int.Parse(speedNeg[i]));
+
+            }
+
+            // Stats
+            t["stats"]["ATTACK_STATS"]["positive"] = node;
+            t["stats"]["ATTACK_STATS"]["neutral"] = node1;
+            t["stats"]["ATTACK_STATS"]["negative"] = node2;
+            t["stats"]["SPEED_STATS"]["positive"] = node3;
+            t["stats"]["SPEED_STATS"]["neutral"] = node4;
+            t["stats"]["SPEED_STATS"]["negative"] = node5;
+
+            // Discord Settings
+            t["discord"]["WEBHOOK_ID"] = boxWebhookID.Text;
+            t["discord"]["WEBHOOK_TOKEN"] = boxWebhookToken.Text;
+            t["discord"]["USER_ID"] = boxUserID.Text;
+            t["discord"]["USER_SHORT_NAME"] = boxPingName.Text;
+            t["discord"]["UPDATE_LEVELS"] = boxPingSettings.Text;
+
+            // Game Language Setting
+            t["language"]["LANGUAGE"] = boxGameLanguage.Text;
+
+            // Pokémon Data Settings
+            t["pokemon_data_paths"]["Boss_Pokemon"] = "data/boss_pokemon.json";
```

Comment:
for all values below, I'd read directly from the config.sample.toml. That way, if there is a change to the config file, no need to update the gui

Reply:
uses the sample to get the language values!